.github/workflows/build_and_test.yml

```diff
@@ -217,42 +222,82 @@ jobs:
           docker image load --input integritee-cli-client-${{ matrix.flavor_id }}-${{ github.sha }}.tar.gz
           docker images --all
 
+      ##
+      # Before tagging, delete the old "stuck" ones to be sure that the newly created ones are the latest
+      # Without if the docker image rmi throws an error if the image doesn't exist.
+      ##
```

Comment:
I find it actually weird that we have to care about this. Are you confident that the GitHub actions runner handle this if it has the correct settings? We used to not need to do this when it wasn't our own runner.

Reply:
We're using the docker engine of the host machine. On very rare occasions it can happen that it stucks.